docker/generic/start_proxy.py

```diff
@@ -605,6 +621,8 @@ def enforce_conflict_args(args):
         return "Flag --ssl_port is going to be deprecated, please use --ssl_server_cert_path only."
     if args.tls_mutual_auth and args.ssl_client_cert_path:
         return "Flag --tls_mutual_auth is going to be deprecated, please use --ssl_client_cert_path only."
+    if args.ssl_client_cert_chain_file and args.enable_grpc_backend_ssl:
+        return "Flag --grpc_backend_ssl_cert_chain_file is going to be deprecated, please use ssl_client_cert_chain_file only."
```

Comment:
return error doesn't match the flag it is checking.

Reply:
updated.